packages/docs/docs/guides/6-hosting-a-node/0-intro.md

```diff
@@ -1,24 +1,50 @@
 ---
+
 title: Introduction to Request Node hosting
 keywords: [Request node]
+---# Introduction
 
----
+Now you should be comfortable with features of the Request network.
 
-# Introduction
+In this guide we will explain what is the Request Node and help you to run your own.
 
-Now you should be comfortable with features of the Request network.
+## What is the Request Node?
+
+Request Nodes are servers that run the lower layers of the Request protocol. They connect to the Ethereum and IPFS networks, to store and retrieve request transactions. Protocol users can interact with the Request Node through HTTP using the [request-client.js](../5-request-client/0-intro) library.
+
+## Why run your own Node?
+
+Running the Node locally on your machine will allow you to test your code using the Request Client easilly.
+
+You may also want to host your Node in a server. Hosting your Node is the most decentralized setup possible. It allows you to:
+
+- Store your own data and make sure it is safelly backed up
+- Be technically independent: own your servers and control how you manage them
+- Use custom configuration settings
+- Customize storage options
+
+## How to run your Node?
+
+There are 3 currently supported ways to run the Request Node:
+
+- Run from [**Docker**](./1-docker). Probably the easiest way to run the Request Node.
+- Run the [**code**](./2-code) from the git repository. Especially useful if you are making changes to the protocol layers.
```

Comment:
`Especially useful if you are making changes to the protocol layers.`
The easiest way to test fast the changes we made is to use docker compose but I don't know which session it should be because we use Docker but in the same way, we run the node from the git repository